### DISCUSSION TITLE
v1.4.0-preview1

### DISCUSSION THREAD

**Discussion by duncanmacmichael** [microsoft/WindowsAppSDK#3737] [Announcements]
## 1.4 Preview 1 (1.4.230628000-preview1) 🔃

The first preview of WinAppSDK 1.4 is now available! Check out the [release notes](https://aka.ms/windowsappsdk/1.4/1.4-preview1/release-notes) for more information on the new & updated features and known issues.

### Updating to this version
The 1.4-preview1 package is available at https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.4.230628000-preview1

### Providing feedback
As we introduce new & upcoming features in our preview releases, we appreciate all of your feedback as we work towards a stable release:
- For WinUI 3 feedback, please open an issue on the [WinUI repo](https://github.com/microsoft/microsoft-ui-xaml/issues/new/choose).
- For feedback regarding all other Windows App SDK components, please open an issue here, on the [Windows App SDK repo](https://github.com/microsoft/WindowsAppSDK/issues/new/choose).

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.4.0-preview1'>v1.4.0-preview1</a>.</em>

**Comment by Slion** [comment]
I like that Flyout fix. I did notice it was a bit funny in my app as it tried to remain on top of the window. Will they also be dismissed when you switch to another app or that's something I need to patch myself? On 1.3 Flyout is closing if you click somewhere else in your app. But if you switch to another app it remains open.
